webpages/settings/index.js

```diff
@@ -650,6 +650,41 @@ document.addEventListener("keydown", (e) => {
 
 chrome.runtime.sendMessage("checkPermissions");
 
+document.getElementById("enableAll").addEventListener("click", () => {
+  Object.values(
+    document.querySelectorAll('.addon-body:not([style="display: none;"]) .switch[state=off]')
+  ).forEach((s) => s.click());
+});
+
+document.getElementById("disableAll").addEventListener("click", () => {
+  Object.values(document.querySelectorAll('.addon-body:not([style="display: none;"]) .switch[state=on]')).forEach((s) =>
+    s.click()
+  );
+});
+
```

Comment:
Use Vue @click, not event listeners for all of these, not just the ones I have selected